src/core/sys/darwin/mach/loader.d

```diff
@@ -159,252 +581,2853 @@ version (CoreDdoc)
+        uint ilocalsym;
+
+        /// Number of local symbols.
+        uint nlocalsym;
+
+
+
+        /// Index into external relocation entries.
+        uint iextrel;
+
+        /// Number of external relocation entries.
+        uint nextrel;
+
+
+
+        /**
+         * Low 16 bits are the index into the init section, high 16 bits are the
+         * index into the term section.
+         */
+        uint iinit_iterm;
+
+        /**
+         * Low 16 bits are the number of init section entries, high 16 bits are
+         * the number of term section entries.
+         */
+        uint ninit_nterm;
+
+
+
+        /**
+         * The (__OBJC,__module_info) section.
+         *
+         * For this module address of the start of.
+         */
+        uint objc_module_info_addr;
+
+        /**
+         * The (__OBJC,__module_info) section.
+         *
+         * For this module size of.
+         */
+        uint objc_module_info_size;
+    }
+
+    /// A 64-bit module table entry.
+    struct dylib_module_64
+    {
+        /// The module name (index into string table).
+        uint module_name;
+
+
+
+        /// Index into externally defined symbols.
+        uint iextdefsym;
+
+        /// Number of externally defined symbols.
+        uint nextdefsym;
+
+        /// Index into reference symbol table.
+        uint irefsym;
+
+        /// Number of reference symbol table entries.
+        uint nrefsym;
+
+        /// Index into symbols for local symbols.
+        uint ilocalsym;
+
+        /// Number of local symbols.
+        uint nlocalsym;
+
+
+
+        /// Index into external relocation entries.
+        uint iextrel;
+
+        /// Number of external relocation entries.
+        uint nextrel;
+
+
+
+        /**
+         * Low 16 bits are the index into the init section, high 16 bits are the
+         * index into the term section.
+         */
+        uint iinit_iterm;
+
+        /**
+         * Low 16 bits are the number of init section entries, high 16 bits are
+         * the number of term section entries.
+         */
+        uint ninit_nterm;
+
+
+
+        /**
+         * The (__OBJC,__module_info) section.
+         *
+         * For this module size of.
+         */
+        uint objc_module_info_size;
+
+        /**
+         * The (__OBJC,__module_info) section.
+         *
+         * For this module address of the start of.
+         */
+        ulong objc_module_info_addr;
+    }
+
+    /**
+     * The entries in the reference symbol table are used when loading the
+     * module (both by the static and dynamic link editors) and if the module is
+     * unloaded or replaced. Therefore all external symbols
+     * (defined and undefined) are listed in the module's reference table. The
+     * flags describe the type of reference that is being made. The constants
+     * for the flags are defined in <mach-o/nlist.h> as they are also used for
+     * symbol table entries.
+     */
+    struct dylib_reference
```

Comment:
is this supposed to have members?

Reply:
No. The C struct contains bitfields. So what I did was to use `std.bitmanip.bitfields` and added the code that `bitfields` expands to. What you have commented here on is a `CoreDdoc` version block. This is at the top to be able to generate documentation on all platforms. Otherwise the file is guarded with `version (Darwin):` and the documentation is not generated on macOS. There is one field in the actual definition of the struct: https://github.com/dlang/druntime/pull/2665/files#diff-3ca62e3adaf537d37782e971d010c0e5R3112.